app/controllers/games_controller.rb

```diff
@@ -17,10 +17,46 @@ def create
       render json: @new_game.errors.details, status: 500
 
     end
+  end
+
+
+  def update
+    current_game = Game.find(params[:id])
+    payload = JSON.parse(request.body.read)
+    position = payload['position'].to_i
+    
+    board = JSON.parse(current_game.board)
+    player = current_game.player
+    toggle = TicTacToeGame::Toggle.new(player)
+
+    game = TicTacToeGame::Game.new(board,player, toggle)
+
+    game.move(position)
+    current_game.player = toggle.current_turn
+    current_game.board = game.board
+    current_player = current_game.current_player
 
+    if game.end?
+      if game.blocked?
+        return render json: "The game ended in a tie", status: 200
+      else
+        return render json: "#{current_player} won the game", status: 200
+      end
+    end
+
+    current_game.current_player =  TicTacToeGame::HumanHumanToggle.new.current_turn(current_player)
+
+    current_game.save()
+
+    render json: game.board, status: 200
+  end
 
+  private
+  def position_params
+    params.require(:game).permit(:board)
   end
 
+  private
```

Comment:
You don't need to have private for each method. Every method below the private keyword is declared as private.